### PR TITLE
Add some debug for OT agent rule loading

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review please?

File: client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingRuleLoader.java [hunk 1]
```diff
@@ -99,6 +103,9 @@ private static void loadRules(URI uri, List<String> scriptNames,
 
                 Files.walk(rules).filter(f -> f.toString().endsWith(RULE_FILE_EXTENSION)).forEach(f -> {
                     try {
+                        if (log.isLoggable(Level.FINE)) {
+                            log.fine("Loading rules: " + f.toString());
```
Comment (pavolloffay) [inline client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingRuleLoader.java]: Does it log full path? or just file name?

Comment (objectiser) [inline client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingRuleLoader.java]: It reports the path relative to /apmrules.

**Comment by objectiser** [comment]
@pavolloffay Thanks!

**Pull request merged**